src/containers/components/contracts/ContractInputConstant.js

```diff
@@ -20,10 +21,11 @@ class ContractInputConstant extends Component {
          </View>
          <View style={styles.btnContainer}>
           <ClearButton
-              buttonText= {`Call ${item.functionSignature}`}
+              buttonText= {`Call ${item.property}`}
               onClickFunction={async () => {
-                const methodResult = await this.props.contractExecution(item.functionSignature);
-                await this.setState({ result: methodResult});
+                const methodResult = await this.props.contractExecution(item);
+								// result has no string which causes crash
```

Comment:
Is this fixed?

Reply:
yes i think so

Reply:
Actually here, methodResult is a bigNumber, which should be printed to UI, so UI loads a line saying **"got big number"**, this should be fixed, according to App needs.